packages/runtime/runtime-utils/src/remoteFluidObjectHandle.ts

```diff
@@ -46,7 +46,9 @@ export class RemoteFluidObjectHandle implements IFluidHandle {
 
     public async get(): Promise<any> {
         if (this.dataStoreP === undefined) {
-            this.dataStoreP = this.routeContext.resolveHandle({ url: this.absolutePath.slice(1) })
+            // Trim leading slash, if it exists
+            const url = this.absolutePath.startsWith("/") ? this.absolutePath.slice(1) : this.absolutePath;
+            this.dataStoreP = this.routeContext.resolveHandle({ url })
```

Comment:
The absolute URL should always start with a "/". If it doesn't we have a deeper problem somewhere. I don't think we should add this check here. I would rather add an assert that the url should start with "/".

Reply:
This is assuming that even if the absolute path doesn't start with "/" it is an absolute path. It will mask cases where the absolute path is actually a relative path.